pkg/dashboard/index.html

```diff
@@ -144,7 +144,7 @@ <h4 class="modal-title" translate>Edit Server</h4>
                                 <input class="form-control" id="host-edit-user"/>
                             </td>
                             <td>
-                                <a tabindex="0" role="button" data-toggle="popover"
+                                <a tabindex="0" tabindex="0" role="button" data-toggle="popover"
```

Comment:
Tabindex was supposed to be removed from the matches, but here I guess I added in an extra one anyway. Whoops!

```suggestion
                                <a tabindex="0" role="button" data-toggle="popover"
```

_**Note**: As this is <a role="button">, we really should make this literally just a <button> (and drop the tabindex too)._